File: find_pivot_index/Find_pivot_index.java
package find_pivot_index;

public class Find_pivot_index {
    public int pivotIndex(int[] nums) {
        var totalSum = 0;
        for (int i = 0; i < nums.length; i++) {
            totalSum += nums[i];
        }
        var startingSum = 0;
        var index = 0;
        while (index < nums.length) {
            if (totalSum - nums[index] - startingSum == startingSum) {
                return index;
            }
            startingSum += nums[index];
            index += 1;
        }
        return -1;
    }
}

File: find_pivot_index/find_pivot_index.py
def pivotIndex(self, nums: List[int]) -> int:
    totalSum = 0
    for num in nums:
        totalSum += num
    startingSum = 0
    index = 0
    while(index < len(numbers)):
        if(totalSum - numbers[index] - startingSum == startingSum):
            return index
        startingSum += numbers[index]
        index += 1
    return -1
